src/views/Dashboard/Customer/Home/MyTest/StakingServiceTab.vue

```diff
@@ -5,56 +5,86 @@
         :headers="headers"
         :items="items"
       )
+        template(v-slot:[`item.country`]="{ item }")
+          div {{ item.country }}
+
+        template(v-slot:[`item.city`]="{ item }")
+          div {{ item.city }}
+
+        template(v-slot:[`item.category`]="{ item }")
+          div {{ item.serviceCategory }}
+
+        template(v-slot:[`item.orderDate`]="{ item }")
+          span 11/11/2021
 
         template(v-slot:[`item.stakeStatus`]="{ item }")
-          span(:style="setButtonBackground(item.stakeStatus)") {{ item.stakeStatus }}
+          span(:style="setButtonBackground(item.status)") {{ getStatusName(item.status)}} 
 
         template(v-slot:[`item.amount`]="{ item }")
-          span(:style="setButtonBackground(item.stakeStatus)") {{ item.amount }}
+          span(:style="setButtonBackground(item.status)") {{ item.stakingAmount/1000000000000000000 }}
```

Comment:
it's better to use web3 or ethers library to convert the staking amount to ether format

Reply:
done, kindly re-review. thanks